### PR TITLE
BugFix - Test of PropertyConverter regarding LotDataEntity

### PR DESCRIPTION
**Pull request by Jugklgl**
bug fixed, all tests pass now

### CODE REVIEW

File: src/test/java/com/example/rets_api/converter/PropertyConverterTest.java [hunk 1]
```diff
@@ -37,7 +37,6 @@ public void dtoTOEntity_PropertyConverter_ShouldReturnCompleteObject(){
         checkAllFields_Room(propertyEntity.getRoomList().get(0), propertyDTO.getRoomList().get(0));
         checkAllFields_School(propertyEntity.getSchoolList().get(0), propertyDTO.getSchoolList().get(0));
         checkAllFields_FinancialData(propertyEntity.getFinancialData(), propertyDTO.getFinancialData());
-        checkAllFields_LotData(propertyEntity.getLotData(), propertyDTO.getLotData());
```
Comment (taisemarques) [inline src/test/java/com/example/rets_api/converter/PropertyConverterTest.java]: why did you removed this?

File: src/test/java/com/example/rets_api/converter/PropertyConverterTest.java [hunk 2]
```diff
@@ -63,7 +62,6 @@ public void entityToDTO_RoomConverter_ShouldReturnCompleteObject(){
...
```
Comment (taisemarques) [inline src/test/java/com/example/rets_api/converter/PropertyConverterTest.java]: why did you removed this?

**Review by taisemarques — CHANGES_REQUESTED** [review]
Need to add the LotData field in Property creation at UtilsTest

File: src/test/java/com/example/rets_api/converter/PropertyConverterTest.java (hunk 1 above)
Comment (Jugklgl) [reply src/test/java/com/example/rets_api/converter/PropertyConverterTest.java]: I added back

File: src/test/java/com/example/rets_api/converter/PropertyConverterTest.java (hunk 2 above)
Comment (Jugklgl) [reply src/test/java/com/example/rets_api/converter/PropertyConverterTest.java]: I added back

**Review by taisemarques — APPROVED** [review]

**Pull request merged**